app/presenters/refund_presenter.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class RefundPresenter < WasteCarriersEngine::BasePresenter
+  def initialize(finance_details, payment)
+    @payment = payment
+
+    super(finance_details, nil)
+  end
+
+  def total_amount_paid
+    @_total_amount_paid ||= payments.inject(0) do |memo, payment|
+      memo + payment.amount
+    end
+  end
+
+  def amount_due
```

Comment:
Could we amend the name to be consistent with the other method?

```suggestion
  def total_amount_due
```